src/modules/Users.pm

```diff
@@ -4561,6 +4571,9 @@ sub Write {
 	    }
 	}
     }
+
+    if (Mode->autoinst() || Mode->autoupgrade() || Mode->config()) { WriteAuthorizedKeys(); }
```

Comment:
You can use `Mode->auto()` which covers both [`autoinst` and `autoupgrade`](https://github.com/yast/yast-yast2/blob/master/library/general/src/modules/Mode.rb#L289) modes.